client/my-sites/site-settings/section-export.jsx

```diff
@@ -2,17 +2,17 @@
  * External dependencies
  */
 import React, { Component } from 'react';
-import i18n from 'i18n-calypso';
+//import i18n from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import EmptyContent from 'components/empty-content';
+//import EmptyContent from 'components/empty-content';
 import ExporterContainer from 'my-sites/exporter';
 
 export default class SiteSettingsExport extends Component {
```

Comment:
Also note that we might be able to wrap this in a higher order component [localize](https://github.com/Automattic/i18n-calypso#localize) that will add translate as a prop.